src/clients.ts

```diff
@@ -0,0 +1,19 @@
+import colors from "./colors";
+
+export const getClientColors = (clientId: string) => {
+  // Naive way of getting an integer out of the clientId
+  const sum = clientId.split("").reduce((a, str) => a + str.charCodeAt(0), 0);
+
+  // Skip transparent background.
+  const backgrounds = colors.elementBackground.slice(1);
+  const strokes = colors.elementStroke.slice(1);
+  return {
+    background: backgrounds[sum % backgrounds.length],
+    stroke: strokes[sum % strokes.length],
+  };
+};
+
+// @TODO: This is terrible for i18n. Also we probably want to get initials if words len > 1?
+export const getClientShortName = (username: string) => {
+  return username.substring(0, 2);
```

Comment:
I'd use initials if user supplies first & last name, and only then fallback to first two chars. Also, let's uppercase the letters in either scenario.

Reply:
Yeah I thought this too but for international names are we ok with doing the same? Does that make sense in all languages? What if a user inputs a middle name (so 3 in total)?

As I don't know these answers I thought a substring might be safer. Uppercase is a good call though. Thoughts?

Reply:
I think it's still safe to go with initials first. If someone uses 3 names (unlikely), we can take first and last. I don't see much harm even if we get it wrong in some edge case, and it'll generally improve the UX.